phoenicis-win32/src/test/java/org/phoenicis/win32/registry/RegistryParserTest.java

```diff
@@ -60,8 +61,8 @@ public void testRegistryParser_parseSimpleFile_testKeyTree() throws IOException
     }
 
     @Test
-    public void testParse_realRegFile_testObjectPopulated() {
-        File registryFile = new File(this.getClass().getResource("user.reg").getFile());
+    public void testParse_realRegFile_testObjectPopulated() throws URISyntaxException {
```

Comment:
Sorry I was thinking that the production (non test) code was actually throwing a checked exception, which is what we are trying to avoid